### ISSUE TITLE
build is broken in Linux and Mac

### ISSUE THREAD

**Issue by joperezr** [dotnet/corefx#7369]
.Net Core MSBuild doesn't recognize characters that were added in 7162d1c516b53e0b4b3b8114b8ce60345d3ecfa7 The error thrown is:
```
HelperTypes.cs(92,15): error CS1056: Unexpected character '�' [/mnt/j/workspace/dotnet_corefx/outerloop_ubuntu14.04_debug/src/System.Linq.Expressions/tests/System.Linq.Expressions.Tests.csproj]
Cast/CastTests.cs(279,71): error CS1056: Unexpected character '�' [/mnt/j/workspace/dotnet_corefx/outerloop_ubuntu14.04_debug/src/System.Linq.Expressions/tests/System.Linq.Expressions.Tests.csproj]
Cast/CastTests.cs(286,71): error CS1056: Unexpected character '�' [/mnt/j/workspace/dotnet_corefx/outerloop_ubuntu14.04_debug/src/System.Linq.Expressions/tests/System.Linq.Expressions.Tests.csproj]
```

@VSadov @JonHanna PTAL

**Comment by stephentoub** [comment]
I already have a PR up to fix it: https://github.com/dotnet/corefx/pull/7365

**Comment by stephentoub** [comment]
Fixed by #7365

**Comment by joperezr** [comment]
Awesome, thanks Stephen
